project/Dependencies.scala

```diff
@@ -2,12 +2,12 @@ import org.scalajs.sbtplugin.ScalaJSPlugin.autoImport._
 import sbt._
 
 object Dependencies {
-
+  // scalastyle:off
   //libs for testing
   lazy val testing: Def.Initialize[Seq[ModuleID]] = Def.setting(Seq(
-    "org.scalatest" %%% "scalatest" % Versions.scalaTest % Test,
+    "org.scalatest" %%% "scalatest" % Versions.scalaTest % Test/*,
 
-    "org.scalatest" %%% "scalatest-matchers" % Versions.scalaTest % Test))
+    "org.scalatest" %%% "scalatest-matchers" % Versions.scalaTest % Test*/))
```

Comment:
Removed the scalatest version in the updated pull request. I think its best that you take a decision on this library version.